Code/GraphMol/MolDraw2D/Wrap/testMolDraw2D.py

```diff
@@ -622,5 +623,69 @@ def testUpdateParamsFromJSON(self):
     txt = d2d.GetDrawingText()
     self.assertTrue('>8</text>' in txt)
 
+  def testIsotopeLabels(self):
+    m = Chem.MolFromSmiles("[1*]c1cc([2*])c([3*])c[14c]1")
+    regex = re.compile("^<text\\s+.*>\\d</text>$")
+    self.assertIsNotNone(m)
+
+    d2d = Draw.MolDraw2DSVG(300, 300, -1, -1, True)
+    d2d.DrawMolecule(m)
+    d2d.FinishDrawing()
+    textIsoDummyIso = d2d.GetDrawingText()
+    nLinesIsoDummyIso = 0
+    for line in textIsoDummyIso.split("\n"):
+      if regex.match(line):
+        nLinesIsoDummyIso += 1
+    self.assertEqual(nLinesIsoDummyIso, 5)
+
+    d2d = Draw.MolDraw2DSVG(300, 300, -1, -1, True)
+    d2d.drawOptions().isotopeLabels = False
+    d2d.DrawMolecule(m)
+    d2d.FinishDrawing()
+    textNoIsoDummyIso = d2d.GetDrawingText()
+    nLinesNoIsoDummyIso = 0
+    for line in textNoIsoDummyIso.split("\n"):
+      if regex.match(line):
+        nLinesNoIsoDummyIso += 1
+    self.assertEqual(nLinesNoIsoDummyIso, 3)
+
+    d2d = Draw.MolDraw2DSVG(300, 300, -1, -1, True)
+    d2d.drawOptions().dummyIsotopeLabels = False
+    d2d.DrawMolecule(m)
+    d2d.FinishDrawing()
+    textIsoNoDummyIso = d2d.GetDrawingText()
+    nLinesIsoNoDummyIso = 0
+    for line in textIsoNoDummyIso.split("\n"):
+      if regex.match(line):
+        nLinesIsoNoDummyIso += 1
```

Comment:
Isn't it simpler and equivalent to switch the regex to:
```
r"<text\s+.*>\d</text>"
```
and then use this form?
```suggestion
    nLinesIsoNoDummyIso = len(regex.findall(textIsoNoDummyIso))
```
The code is simpler *and* shorter and you're actually testing what you want to test. (The current regex will break for reasons unrelated to what you're actually testing if we, for whatever reason, stop putting each `<text>` block on its own line)